### ISSUE TITLE
Ambassador Diagnostics Interface Reporting Newer Version Available When There Isn't

### ISSUE THREAD

**Issue by WillPlatnick** [datawire/ambassador#1807]
Go to the Diagnostics UI

Ambassador version 0.77.0
displays
INFO: Upgrade available! to Ambassador version 0.77.0

**Comment by ppeble** [comment]
We are seeing this as well. From the logs:

```
2019-09-10 21:12:32 diagd 0.77.0 [P91TAmbassadorEventWatcher] INFO: Scout reports {"latest_version": "0.77.0", "application": "ambassador", "cached": false, "timestamp": 1568149952.927258}
2019-09-10 21:12:32 diagd 0.77.0 [P91TAmbassadorEventWatcher] INFO: Scout notices: [{"level": "INFO", "message": "Upgrade available! to Ambassador version 0.77.0"}]```

**Comment by Aracki** [comment]
I was facing the similar issue.
